client/extensions/woocommerce/state/wc-api/reducer.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Next time, instead of cutting and pasting here, you could just base your branch on the branch upon which you depend.